CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ## Changed
 - [[#234](https://github.com/rust-vmm/kvm-ioctls/issues/234)] vcpu: export
-reg_size as a public method.
+  reg\_size as a public method.
```

Comment:
Accidental change?

Reply:
No, intentional fix up, because the underscore needs to be escaped in markdown (otherwise it can lead to unintended cursive text)

Reply:
I think this should be in its own commit (can still be this PR).